modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -474,6 +488,89 @@ You can test whether the service is available at `_API_PLATFORM_HOST_/apiplatfor
 |===
 
 
+[[encrypt_password]]
+== Encrypt Passwords in an Existing mule-agent.yml File
+
+
+If you didn't set the `AGENT_VAR_master_password` environment variable when registering the agent 
```

Comment:
"environment variable" = "these passwords"?
Just looks odd, singular to plural.

Reply:
A single env variable encrypts all passwords in the file.